niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/SignaturePermissionTester.java

```diff
@@ -3361,17 +3558,21 @@ public void writeToParcel(Parcel parcel, int i) {
                             mPackageName);
                 }));
 
+        //This permission's category has been moved to install after Android T
+        //The test leave it as is for the previous releases.
         mPermissionTasks.put(permission.READ_NEARBY_STREAMING_POLICY,
                 new PermissionTest(false, Build.VERSION_CODES.S, () -> {
```

Comment:
If the category for this permission was moved to install for T, should this have Sv2 as the max SDK version? Otherwise on T+ devices, when the permission is not granted, this method would just complete without an exception and result in a test failure?

Reply:
Thanks, added max SDK version. I think signature permission test would be execute with AndroidManifest.xml whicn granting all of the permssions, so the test will succeed.

Reply:
I agree, most likely this would not cause any failures, although it may be confusing if the test shows up under both the install and signature permission tests on devices running Android T+. The change looks good to me, thanks!